auth/src/main/java/org/aerogear/mobile/auth/AuthService.java

```diff
@@ -34,7 +34,6 @@
 public class AuthService implements ServiceModule {
     private final static Logger LOG = MobileCore.getLogger();
     private final static String TAG = "AuthService";
```

Comment:
Should be `static final` instead of `final static` (see https://docs.oracle.com/javase/specs/jls/se7/html/jls-8.html#jls-8.3.1)